server/pulp/server/managers/consumer/agent.py

```diff
@@ -14,10 +14,13 @@
 """
 
 import sys
+
 from logging import getLogger
 from uuid import uuid4
+from gettext import gettext as _
```

Comment:
On 01/29/2015 10:29 AM, Jeff Ortel wrote:
> alphabetical? I'm not aware of this requirement in pep8. can you point
> me to the text?

Agreed with what @bmbouter said, and I'll add that it's specifically
helpful because it groups imports from the same module and nearby
modules together.